orttraining/orttraining/test/optimizer/graph_transform_test.cc

```diff
@@ -13,7 +13,7 @@
 #include "core/optimizer/bias_gelu_fusion.h"
 #include "core/optimizer/gelu_fusion.h"
 #include "core/optimizer/dropout_elimination.h"
-#include "orttraining/core/optimizer/bias_dropout_fusion.h"
+#include "core/optimizer/bias_dropout_fusion.h"
```

Comment:
>"core/optimizer/bias_dropout_fusion.h" [](start = 9, length = 38)

should probably move the tests for BiasDropoutFusion to onnxruntime/test/optimizer/graph_transform_test.cc so they run when training is not enabled

Reply:
moved.